### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by fzhuang3** [penghu-cs/DSCMR#4]
It seems your data split follows "Cross-modal Retrieval with Correspondence Autoencoder", am I right?

Can I ask how the validation set was used? In your code here the test set is used for validation. Am I right to say the validation set is ignored?

**Comment by masaaldosey** [comment]
> It seems your data split follows "Cross-modal Retrieval with Correspondence Autoencoder", am I right?
> 
> Can I ask how the validation set was used? In your code here the test set is used for validation. Am I right to say the validation set is ignored?

further, is there any early stopping carried out? or was the model trained for 500 epochs and then the metrics evaluated on the test set?
